geometric/run_json.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+import json
+import geometric
+
+def get_qc_schema_traj(qc_schema_input, progress):
+    qc_schema_traj = []
+
+    return qc_schema_traj
+
+def parse_json_input(in_json):
+    """
+    Parse an input json file into options, example:
+    {
+        "schema_name": "qc_schema_optimization_input",
+        "schema_version", 1,
+        "geometric_options": {
+            "coordsys": "tric",
+            "conv": 1.e-7
+        }
+        "input_specification": qc_schema_input,
+    }
+
+    qc_schema_input = {
+        "schema_version": "v0.1",
+        "molecule": {
+            "geometry": [
+                0.0, 0.0, -0.1294769411935893, 0.0,
+               -1.494187339479985, 1.0274465079245698,
+                0.0, 1.494187339479985, 1.0274465079245698
+            ],
+            "symbols": ["O", "H", "H"],
+            "connectivity": [[0, 1, 1], [0, 2, 1]]
+        },
+        "driver": "gradient",
+        "model": {
+            "method": "UFF",
+            "basis": None
+        },
+        "keywords": {},
+        "program": "rdkit"
+    }
+    """
+    d = json.load(open(in_json))
+    input_opts = d['geometric_options']
+    # insert `fix_orientation` and `fix_com`
+    d['input_specification'].update({
+        'fix_orientation': True,
+        'fix_com': True
```

Comment:
Here `fix_orientation` and `fix_com` are forced to True according to @dgasmith 's suggestion.

Reply:
These fixes need to be applied to the `molecule` specification so that programs do not align them to internal representations.

Reply:
Thanks! This has been fixed.